apps/dsb-client-gateway-api/src/app/modules/message/service/dsb-message-pooling.service.ts

```diff
@@ -45,147 +58,252 @@ export class DsbMessagePoolingService implements OnModuleInit {
     const callback = async () => {
       await this.handleInterval();
     };
-    const timeout = setTimeout(callback, this.configService.get<number>('WEBSOCKET_POOLING_TIMEOUT', 5000));
+    const timeout = setTimeout(
+      callback,
+      this.configService.get<number>('WEBSOCKET_POOLING_TIMEOUT', 5000)
+    );
     this.schedulerRegistry.addTimeout(SCHEDULER_HANDLERS.MESSAGES, timeout);
 
     this.logger.log('Enabling websockets');
-
   }
 
   @Span('ws_pool_messages')
   public async handleInterval(): Promise<void> {
     const callback = async () => {
       // handling callback polling msg
-      this.logger.log("[handleInterval] handling callback polling msg");
-      await this.handleInterval();
+      this.logger.log('[handleInterval] handling callback polling msg');
+      const store = new Store(this.pinoLogger.logger);
+
+      await storage.run(store, async () => {
+        const runId: string = uuidv4();
+
+        this.pinoLogger.assign({
+          runId,
+        });
+
+        this.logger.log(`run id ${runId}`);
+
+        await this.handleInterval();
+      });
     };
 
     try {
-      this.logger.log("[deleteTimeout] Start deleteTimeout");
+      this.logger.log('[deleteTimeout] Start deleteTimeout');
       this.schedulerRegistry.deleteTimeout(SCHEDULER_HANDLERS.MESSAGES);
-      this.logger.log("[deleteTimeout] End deleteTimeout");
-
-      if (this.websocketMode === WebSocketImplementation.SERVER && this.gateway.server.clients.size === 0) {
-        const timeout = setTimeout(callback, this.configService.get<number>('WEBSOCKET_POOLING_TIMEOUT', 5000));
+      this.logger.log('[deleteTimeout] End deleteTimeout');
+
+      if (
+        this.websocketMode === WebSocketImplementation.SERVER &&
+        this.gateway.server.clients.size === 0
+      ) {
+        const timeout = setTimeout(
+          callback,
+          this.configService.get<number>('WEBSOCKET_POOLING_TIMEOUT', 5000)
```

Comment:
Set this to a variable at the start, it's repeated a lot, and I don't believe it can change without restarting the app anyway

Reply:
I agree but I made only stylish changes, will enforce linter about it